src/main/resources/data/minecolonies/researches/civilian/1_1_stamina.json

```diff
@@ -0,0 +1,16 @@
+{
+  "comment": "name: Stamina * description : Hospital",
+  "id": "stamina",
+  "branch": "civilian",
+  "requiredUniversityLevel": 1,
+  "icon": "minecolonies:blockhuthospital",
+  "exclusiveChildResearch": true,
```

Comment:
can you run the idea auto formatter on the jsons only?

Reply:
Yeah. Sorry about those, the auto generate tool I’ve got was originally for a different context